### DISCUSSION TITLE
Need some help with the paramaters of the script

### DISCUSSION THREAD

**Discussion by ovizii** [KBlixt/subcleaner#55] [Q&A] [answered]
I was planning to do a dry run and only log deletions, so I could manually go through the log and submit false positives.

Unfortunately, the log either contains nothing when using these parameters: 

` python3 /subcleaner/subcleaner.py -n -a -e -r /mnt/movies/`

Or just about everything from info to warnings with these parameters:

` python3 /subcleaner/subcleaner.py -n -a -r /mnt/movies/`

Looking for the right parameters to do a dry-run and manually go through the logs, so I only want to see deletions with enough details to submit a false positive report if necessary.

**Comment by KBlixt** [comment, marked as answer]
-a -n -r

Is the best option for this I think.

The -a option makes it only print removed blocks in the report. But there are still other logging appearing between the reports.

I could make an option for this in the future. That's a good idea. 

But if you want a workaround for now you could use something like grep to grep on any line starting with a whitespace. This would remove all the lines that contain the other logging info.

Any false positive reports are greatly appreciated 👍

**Reply by ovizii** [reply]
I've run a dry run on my media files and have then filtered down the logs with:

`grep -E "INFO: now cleaning subtitle|^ +" logfile.log > filtered_logfile.log`

I'll submit false positives once I've gone through the logs.
